config/master.d/50-master.conf

```diff
@@ -13,6 +13,12 @@ timeout: 20
 file_roots:
   base:
     - /usr/share/salt/kubernetes/salt
+
 pillar_roots:
   base:
     - /usr/share/salt/kubernetes/pillar
+
+ext_pillar:
+  - velum:
+      url: https://localhost/internal-api/v1/pillar.json
+      ca_bundle: /etc/pki/ca.crt
```

Comment:
What editor do you use? I'm always confused with this "No newline at the enf of file"

Reply:
I use the TRUE editor, best known as emacs.